playbooks/cloud-provider-openstack-acceptance-test-lb-octavia/run.yaml

```diff
@@ -55,15 +55,17 @@
       shell: docker login -u {{ dockerhub.username }} -p {{ dockerhub.password }}
       no_log: yes
 
-    - name: Build openstack-cloud-controller-manager image
+    - name: Build and upload openstack-cloud-controller-manager image
       shell:
-        cmd: make image-controller-manager REGISTRY={{ dockerhub.username }} VERSION={{ zuul.change }}
+        cmd: |
+          export BUILD_CMDS=openstack-cloud-controller-manager
+          export IMAGE_NAMES=openstack-cloud-controller-manager
+          export ARCHS=${ARCHS:-amd64}
+
+          REGISTRY={{ dockerhub.username }} VERSION={{ zuul.change }} make upload-images
```

Comment:
In the CPO Makefile, we have 
```
docker login -u="$(DOCKER_USERNAME)" -p="$(DOCKER_PASSWORD)";
```
Do we need to specify those two vars as well?

Reply:
Good point, I already send a patch to correct that behavior for gate in https://github.com/kubernetes/cloud-provider-openstack/pull/967
but that one can only merge after we fix the gate here. : (
Will update with login info